Sources/MapboxMaps/Style/Types/ExpressionArgumentBuilder.swift

```diff
@@ -90,3 +91,16 @@ extension UIColor: ExpressionArgumentConvertible {
         return [.string(StyleColor(self).rgbaString)]
     }
 }
+
+extension GeoJSONObject: ExpressionArgumentConvertible {
+    public var expressionArguments: [Expression.Argument] {
+        do {
+            let data = try JSONEncoder().encode(self)
+            let expression = try JSONDecoder().decode(Expression.Argument.self, from: data)
+            return [expression]
+        } catch {
+            // Throwable getters are new to Swift 5.5, so we’ll have to live with a fatal error for now.
+            fatalError("Unable to convert GeoJSON object to expression argument: \(error)")
+        }
```

Comment:
I think this can just be:
```suggestion
        return [.geoJSONObject(self)]
```